src/main/java/io/appium/java_client/ios/IOSDriver.java

```diff
@@ -172,6 +173,24 @@ public RequiredElementType findElementByIosUIAutomation(String using)
         throws WebDriverException {
         return (List<RequiredElementType>) findElements("-ios uiautomation", using);
     }
+    
+    /**
+     * @throws org.openqa.selenium.WebDriverException This method is not applicable with browser/webview UI.
+     */
+    @SuppressWarnings("unchecked") @Override
+    public RequiredElementType findElementByIosNsPredicate(String using)
+    throws WebDriverException {
+        return (RequiredElementType) findElement("predicate string", using);
```

Comment:
I'd prefer for the locator strategy to be called `"-ios ns predicate"` or `"-ios predicate string"`. either way it must have a `-` in front of it (see `"-ios uiautomation"` for analogy above)